com.github.fabiocolacio.marker.json

```diff
@@ -0,0 +1,60 @@
+{
+  "app-id": "com.github.fabiocolacio.marker",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "/bin/com.github.fabiocolacio.marker",
```

Comment:
It should be just a filename, not full path.

Reply:
Fixxed in https://github.com/flathub/flathub/pull/249/commits/49f949ce2e39c1035d4b4a9a308827452a8dc6ae